cookie-store/cookieStore_set_arguments.https.any.js

```diff
@@ -285,3 +285,29 @@ promise_test(async testCase => {
+promise_test(async testCase => {
+  // The maximum attribute value size is specified at https://wicg.github.io/cookie-store/#cookie-maximum-attribute-value-size.
+  const maximumAttributeValueSize = 1024
+  await cookieStore.delete('cookie-name');
+
+  await promise_rejects_js(testCase, TypeError, cookieStore.set(
+      { name: 'cookie-name',
+        value: 'cookie-value',
+        path: '/this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-/' }));
+  const cookie = await cookieStore.get('cookie-name');
+  assert_equals(cookie, null);
+}, 'cookieStore.set checks if the path is too long');
+
+promise_test(async testCase => {
+  // The maximum attribute value size is specified at https://wicg.github.io/cookie-store/#cookie-maximum-attribute-value-size.
+  const maximumAttributeValueSize = 1024
+  await cookieStore.delete('cookie-name');
+
+  await promise_rejects_js(testCase, TypeError, cookieStore.set(
+      { name: 'cookie-name',
+        value: 'cookie-value',
+        domain: '/this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-/' }));
```

Comment:
Make this a valid domain, so the only error is the length limit.

Reply:
Good point. Fixed!